poreCov.nf

```diff
@@ -416,8 +417,8 @@ ${c_yellow}Parameters - nCov genome reconstruction${c_reset}
     --primerV       artic-ncov2019 primer_schemes [default: ${params.primerV}]
                         Supported: V1, V2, V3, V1200
     --rapid         use rapid-barcoding-kit [default: ${params.rapid}]
-    --minLength     min length filter raw reads [default: ${params.minLength}]
-    --maxLength     max length filter raw reads [default: ${params.maxLength}]
+    --minLength     min length filter raw reads [default: 350; 500 for V1200 primer scheme]
```

Comment:
@DataSpott `[default: 350; 500 for V1200 primer scheme]` to `[default: 350; 500 for V3; V1200 primer scheme]`

Reply:
applied a change to make it better understandable